syntax = "proto3";

enum COLOR {
  COLOR_BLUE = 0;
}

enum COLORV2 {
  COLORV2_BLUE = 0;
}

message ABTestParam {
  string key = 1;
  string value = 2;
}

message AbKeys {
  repeated ABTestParam all_ab_keys = 1;
  repeated ABTestParam used_ab_keys = 2;
}

message RsRequest {
  AbKeys abKeys = 16;
}